code/game/turfs/exterior/_exterior.dm

```diff
@@ -60,8 +60,12 @@
 		set_light(owner.lightlevel, 0.1, 2)
 		if(owner.planetary_area && istype(loc, world.area))
 			ChangeArea(src, owner.planetary_area)
-	update_icon(TRUE)
+	..()
+	. = INITIALIZE_HINT_LATELOAD
+
+/turf/exterior/LateInitialize()
 	. = ..()
+	update_icon(TRUE)
```

Comment:
Couldn't you do this by calling queue icon update in Initialize instead?

Reply:
The argument needs to be passed and it's important that it happens late, I wasn't aware I could give arguments to the queue proc.